hoot-core/src/main/cpp/hoot/core/algorithms/changeset/ChangesetDeriver.cpp

```diff
@@ -162,7 +176,10 @@ Change ChangesetDeriver::_nextChange()
       LOG_VART(_fromE);
     }
 
-    if (_allowDeletingReferenceFeatures)
+    LOG_VART(_fromE->getVersion());
+    LOG_VART(_fromE->getStatus());
+    if (_allowDeletingReferenceFeatures &&
+        !_fromE->getTags().contains(MetadataTags::HootChangeExcludeDelete()))
     {
```

Comment:
Added the capability to allow excluding elements from delete statements if they are tagged a certain way